README.md

```diff
@@ -1,2 +1,258 @@
 # serpens
+
 A set of Python utilities, recipes and snippets
+
+- [SQS Utilities](#sqs-utilities)
+- [API Utilities](#api-utilities)
+- [Schema](#schema)
+- [CSV Utils](#csv-utils)
+- [DynamoDB Documents](#dynamodb-documents)
+
+## SQS Utilities
+
+- This utility is a decorator that iterate by each sqs record.
+
+- For each sqs record will be inserted de *record object (from type sqs.Record)* as  argument that will to process the sqs messages.
```

Comment:
Typo: `inserted de *record`, acho que você quis dizer "the" em vez de "de"

"will to process" acho que tá errado tb. O correto seria "will process", não?

Reply:
@rodrigosantiag o PR foi atualizado com as sugestões acima.